first_hour_on_vwb/working_with_data.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
The word "Data" can mean a lot of things so I wonder if we should be a bit more accurate with the name of this notebook. `working_with_data_references`? `working_with_referenced_resource_data`? `working_with_bq_resources`?

Reply:
renamed!